### PR TITLE
Update from Python 3.8 to Python 3.9

### PR DESCRIPTION
**Pull request by areski**
## Description

Upgrade from Python 3.8 to 3.9

Checklist:

- [X] I've made sure that `tests/test_cookiecutter_generation.py` is updated accordingly (especially if adding or updating a template option)
- [X] I've updated the documentation or confirm that my change doesn't require any updates

## Rationale

Most of peoples starting a project with `cookiecutter-django` would want to use a more recent version of Python.

### CODE REVIEW

**Comment by foarsitter** [comment]
Nice enhancement, trying it locally now. 

Is it a good idea to take #1810 into account?

**Comment by areski** [comment]
@foarsitter django-phonenumber-field doesn't seem to be used here or I might have missed it. 
Could you link to where this might be?

No doubt about the usefulness of #1810, but this PR was not trying to address this.

**Comment by foarsitter** [comment]
You are right about that `django-phonenumber-field` is not used in this project.

**Comment by browniebroke** [comment]
Nice work @areski! Changes look good. We'll need to test a few prod deployment, as some libraries are not used in test.

For whoever is interested to see progress on this pull request, they can help by testing it with their favorite setup and reporting back...

> No doubt about the usefulness of #1810, but this PR was not trying to address this.

I agree that we should keep #1810 separate 👍

Thanks again for picking this up.

File: {{cookiecutter.project_slug}}/runtime.txt [hunk 1]
```diff
@@ -1 +1 @@
-python-3.8.7
+python-3.9.1
```
Comment (Andrew-Chen-Wang) [inline {{cookiecutter.project_slug}}/runtime.txt:1]: I believe we're on 3.9.2 now.

**Review by Andrew-Chen-Wang — COMMENTED** [review]
@browniebroke Any updates on this PR?

**Comment by browniebroke** [comment]
Thanks for the change! 🎉

**Comment by areski** [comment]
Thanks @browniebroke

**Pull request merged**